SimplilearnSecond.iml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What does this file do? Is it really needed? Wanna do some research 🔍 ?

Reply:
It seems, there are 'modules' in Intellij, and those are created at the same time you create the projects, so this .iml file saves the configuration, dependencies and so on things related to that module. Because, according to what I read you can have more than one module, and then grouped or hide them as you nee, that's why that files exists. .... Interesting though (did I use this well? 😆 )...

Reply:
Did you use what exactly?

Reply:
Yes, it seems that it stores part of your project's configuration, but: [why is it on the root?](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206878055-Why-is-the-iml-file-stored-in-project-root-) 

I am guessing that while trying to get rid from the `.idea` folder from Git, the IDE created the project configuration on the root folder, because it didn't find the `.idea` one. This file is only useful for Carolina, but not for anybody else in the project, Diana might be using Eclipse, I think she is kind of old fashioned.

Remove it from the codebase.

Reply:
Oki, you're right. It happened to me at the job, I created a CR with an update that I did, and it contained a large html file, that was introduce by me, I don't have idea where that came from, and I was not aware of it until someone else ask me what was that :| shame... I will be more careful from now on. Thanks :)

Reply:
> xactly?

the word 'though'

Reply:
Yes it is correctly used.

Reply:
> Yes, it seems that it stores part of your project's configuration, but: [why is it on the root?](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206878055-Why-is-the-iml-file-stored-in-project-root-)
> 
> I am guessing that while trying to get rid from the `.idea` folder from Git, the IDE created the project configuration on the root folder, because it didn't find the `.idea` one. This file is only useful for Carolina, but not for anybody else working the project, Diana might be using Eclipse, I think she is kind of old fashioned.
> 
> Remove it from Git.

hahahhah, do you really think that Diana is old fashioned? 😆

Reply:
I think so, a bit sometimes, she is full of surprises as well.